### PR TITLE
update to new react-native-mp-android-chart base on mpAndroidChart V3

### PR DESCRIPTION
**Pull request by wuxudong**
update to new react-native-mp-android-chart base on mpAndroidChart V3

### CODE REVIEW

File: app/BubbleChartScreen.js [hunk 1]
```diff
@@ -37,36 +37,35 @@ class BubbleChartScreen extends React.Component {
...
-            xValues: _.map(_.range(size), (val) => val.toString())
           }
         }
       })
     );
   }
 
   _randomYValues(range: number, size: number) {
-    return _.times(size, () => {
+    return _.times(size, (index) => {
```
Comment (mskec) [inline app/BubbleChartScreen.js]: index is not used

File: package.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
   },
   "dependencies": {
     "lodash": "^4.12.0",
-    "react": "15.3.2",
+    "react": "15.4.2",
     "react-addons-update": "15.3.2",
-    "react-native": "0.33.0",
-    "react-native-mp-android-chart": "0.2.0"
+    "react-native": "^0.41.2",
```
Comment (mskec) [inline package.json:12]: Please lock vesrsions

**Review by mskec — CHANGES_REQUESTED** [review]
Thanks for updating example.

I made some screenshots with some problems I discovered http://imgur.com/a/qXjbg. In image description you can see what is not like before. If you download current .apk from this repo you can see how it was before.

Additionally BubbleChart, animations are not working as before. Do you know why? Also default zoom is little different.

**Comment by wuxudong** [comment]
for the bar chart, additional config required.

 granularityEnabled:true,
 granularity:1

I will update examples later

**Comment by wuxudong** [comment]
update again.

Because it depends on react-native-mp-android-chart whose pr is under review. you should update the package.json 

`"react-native-mp-android-chart": "file:../the path of react-native-mp-android-chart/"` to test it.

if react-native-mp-android-chart‘s new version is released, then you can update it to correct version number.

File: app/TimeSeriesLineChartScreen.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ class TimeSeriesLineChartScreen extends React.Component {
         fontStyle: 1,
         custom: {
           colors: ['red', 'blue'],
-          labels: ['REFER', 'USER',]
+          // labels: ['REFER', 'USER',]
```
Comment (mskec) [inline app/TimeSeriesLineChartScreen.js]: what about this?

File: app/TimeSeriesLineChartScreen.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+              }
+            }, {
+              values: [{x: 5, y: 90},
+                {x: 10, y: 130},
+                {x: 50, y: 2000, marker: "eat more"},
+                {x: 80, y: 9000, marker: "your are overweight, eat less"}],
+              label: 'user',
+              config: {
+                lineWidth: 1,
+                drawCubic: true,
```
Comment (mskec) [inline app/TimeSeriesLineChartScreen.js:80]: update to `mode`

**Review by mskec — COMMENTED** [review]
Could you update BarChart with additional config so that it's the same as before?

**Pull request merged**